src/upgrade-db-to-2.15.0-postgres.sql

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Our next release is 2.14.1. Do you want this feature to be included in the next release? If you do, you need to rename the file to 2.14.1

Reply:
This file has been renamed